tests/run-make/pgo-branch-weights/rmake.rs

```diff
@@ -9,6 +9,7 @@
 
 //@ needs-profiler-support
 //@ ignore-cross-compile
+//@ ignore-windows-gnu
```

Comment:
Suggestion: we can help future contributors understand tests better if we explain why a test is `ignored-<filter>`. In this case, something like

```rs
//@ ignore-windows-gnu (have problems generating profdata, could use further investigation)
```

would help test readers a lot.

Reply:
By the way, the problem in this case is that the profiler runtime is just broken on mingw. See #122613 for more context.